src/test/java/org/broadinstitute/hellbender/utils/hmm/HMMUnitTest.java

```diff
@@ -69,19 +65,12 @@
 
     private static List<Pair<List<TestHMM.State>, List<TestHMM.Datum>>> TEST_SEQUENCES;
 
-    private static File TEST_SEQUENCE_FILE;
-
-    private static File TEST_R_RESULTS_FILE;
+    private static final File TEST_R_RESULTS_FILE = new File(packageRootTestDir, "utils/hmm/hmm-unit-test-truth.tsv");
 
     private static List<ExpectedResult> TEST_EXPECTED_RESULTS;
 
-    @Test
-    public void testSetup() throws IOException {
-        setUp();
-    }
-
     /**
-     * This method runs the Test models and sequences against R's HMM package to
+     * This method runs the Test models and sequences against results generated using R's HMM package to
```

Comment:
The R package was only used to generate results for this test code and is not used in production code, so I'd rather not install it if necessary.  For that matter, even the production code that this tests is not in use, we just kept it around in case anyone needed a pure Java HMM in the future.

I can try to clean up this test class a bit though.